.github/ISSUE_TEMPLATE/fedora-ci.yml

```
@@ -0,0 +1,32 @@
+name: Packit as a Fedora dist-git CI issue
+description: Report a bug/request a feature for Packit as a Fedora dist-git CI
```

```suggestion
description: Report a bug or request a feature for Packit as a Fedora dist-git CI
```
It was hard for me to read the line with the /. But feel free to ignore it.